crazy_miner/telemedicin_models.py

```
@@ -342,21 +342,28 @@ class CrazyMinerPayment(models.Model):
         ('cancelled', 'لغو شده'),
     ]
     
+    PAYMENT_TYPE_CHOICES = [
+        ('wallet_charge', 'شارژ کیف پول'),
+        ('service_payment', 'پرداخت خدمات'),
+    ]
+    
     id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
```

Ruff (0.12.2)

Undefined name `uuid`

(F821)